crates/services/txpool/src/txpool.rs

```diff
@@ -63,6 +67,10 @@ where
         }
     }
 
+    pub fn config(&self) -> &Config {
```

Comment:
You don't need it anymore

Reply:
I turned it into test-only method since it was used in the tests in other files